test/test_other.py

```diff
@@ -13390,6 +13390,49 @@ def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
 
+  @parameterized({
+    'memory64_wasm_function': (True, True),
+    'wasm_function': (False, True),
+    'memory64': (True, False),
+    '': (False, False)
+  })
+  def test_add_js_function_64_bit(self, memory64, wasm_function):
```

Comment:
We have a test in test_core.py called `test_add_function`.   I wonder if we should call this one `test_add_function_bigint` and place it alongside the other one?

(Not sure why that other one lives in `test/interop` all on its own..

Reply:
I wasn't sure if we wanted to do it with all the 100 variations of optflags. I don't think the logic tested here is particularly sensitive to optflags. But I can move it there if you prefer.

Reply:
Could you perhaps rename it to `test_add_function_bigint` and a comment/TODO refering to `test_add_function` in test_core.py?

It seems like those two tests should probably live together, if not now then at some point.

Reply:
Are you opposed to moving test_add_function to test_other.py?

Reply:
Okay I moved `test_add_function` to `test_other.py`, marking this as resolved but let me know if you want me to revert and add comments saying these should live together instead.